src/style/main.css

```diff
@@ -185,7 +206,7 @@ a:visited {
 
 @media (max-width: 960px) {
   #app {
-    flex-direction: column;
+    flex-direction: column-reverse;
```

Comment:
isn't this less accessible? DOM is different from what you see?

Reply:
The DOM order is:

- top app bar
- menu on the side
- map

But on mobile, we want the menu to visually stick to the bottom, and in order to achieve that we need to column-reverse reverse it. Maybe @erik has some comments?